engine/src/main/java/org/destinationsol/game/ship/FarShip.java

```diff
@@ -189,4 +190,24 @@ public float getMoney() {
     public ItemContainer getIc() {
         return container;
     }
+
+    /**
+     * Each FarShip could be a mercenary. Each mercenary is definitely a FarShip .
```

Comment:
This really should not be a part of method-level javadoc, but rather a class-level javadoc. Also, javadoc should start with a single sentence summing the shole rest of the javadoc, ended by a dot and separated from the rest by a blank line.

Reply:
Wait. Now I got to the end of the changes and... 
1) Fine, I'll let the javadoc formatting slip for GhostInTheMachine
2) Your javadoc doesn't really make sense, and the SolShip's should be changed too